src/actors/builtin/storage_miner/storage_miner_actor_code.go

```diff
@@ -59,12 +59,8 @@ func (a *StorageMinerActorCode_I) OnSurprisePoStChallenge(rt Runtime) {
 		numConsecutiveFailures = st.PoStState().As_DetectedFault().NumConsecutiveFailures()
 	}
 
-	IMPL_TODO() // Determine auxiliary seed input to ensure uniqueness
-	challengedSectorsRandomness := rt.RandomnessWithAuxSeed(
-		filcrypto.DomainSeparationTag_SurprisePoStSampleSectors,
-		rt.CurrEpoch()-node_base.SPC_LOOKBACK_POST,
-		addr.Serialize_Address_Compact(rt.CurrReceiver()),
-	)
+	randomnessK := rt.GetRandomness(rt.CurrEpoch() - node_base.SPC_LOOKBACK_POST)
```

Comment:
Removing the other seed parameters seems error-prone here, as it encourages users of the API to make calls with potentially duplicated inputs.

Reply:
It's a good point. I thought about it from the standpoint of separating where you get your random seed from and how you compose it to get the wanted random input. will look at refactoring (it's been a bit so wanna convince myself i should)

Reply:
I like the decomposition - the code behind the runtime API should be as minimal as possible. Perhaps some type definition could provide the prompt to then combine that with other information. Call the thing from the chain a randomness seed.